_posts/2025-03-11-release-2.9.0.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: post
+title: Release 2.9.0 of MDAnalysis
+---
+
+We are happy to release version 2.9.0 of MDAnalysis!
+
+This is a minor release of the MDAnalysis library, which means that it
+contains enhancements, bug fixes, deprecations, and other
+backwards-compatible changes.
+
+
+## Supported environments
+
+This release supports NumPy 2.0+ and offers backwards compatibility
+through to NumPy 1.23.2.
+
+Supported Python versions: **3.10, 3.11, 3.12, 3.13**.
+
+Please note that Python 3.13 is limited to [PyPi][] for now, the [conda-forge channel][] installs only provide support for Python 3.10 to 3.12.
```

Comment:
@hmacdope @IAlibay at the moment there are no py3.13 c-f packages https://anaconda.org/conda-forge/mdanalysis/files?page=1 so we have to keep this line (see https://github.com/MDAnalysis/mdanalysis/issues/4805 ).

Is there a chance that this gets fixed soon (within a day) or should we publish the post as is?

Reply:
I'm travelling sorry, it's a simple migration if others have time though.

Reply:
@hmacdope do you know what to do?

Reply:
Thank you both, we now have 3.13 packages.